README.md

```diff
@@ -96,19 +96,20 @@ submodel = model.Submodel(identifier)
 ```
 
 Create a `Property` and add it to the `Submodel`:
+
 ```python
 # create a global reference to a semantic description of the property
-semantic_reference = model.GlobalReference(
-    (model.Key(
-        type_=model.KeyTypes.GLOBAL_REFERENCE,
-        value='http://acplt.org/Properties/SimpleProperty'
-    ),)
+semantic_reference = model.ExternalReference(
+  (model.Key(
+    type_=model.KeyTypes.GLOBAL_REFERENCE,
+    value='http://acplt.org/Properties/SimpleProperty'
+  ),)
 )
 property = model.Property(
-    id_short='ExampleProperty',  # Identifying string of the element within the submodel namespace
-    value_type=model.datatypes.String,  # Data type of the value
-    value='exampleValue',  # Value of the property
-    semantic_id=semantic_reference  # set the semantic reference
+  id_short='ExampleProperty',  # Identifying string of the element within the submodel namespace
+  value_type=model.datatypes.String,  # Data type of the value
+  value='exampleValue',  # Value of the property
+  semantic_id=semantic_reference  # set the semantic reference
```

Comment:
Why have these lines been re-indented with 2 spaces instead of 4?

Reply:
I have honestly no idea. Must have been PyCharm during rebasing